frontend/src/component/common/TimeAgo/TimeAgo.tsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState, type FC } from 'react';
+import { formatDistanceToNow, secondsToMilliseconds } from 'date-fns';
+
+type TimeAgoProps = {
+    date: Date | number | string | null | undefined;
+    fallback?: string;
+    live?: boolean | number;
+};
```

Comment:
I was a little confused by the type here, because it serves two purposes, right? Either it's "auto-update for me", which is well described by live, or it's "update every n seconds", which is not very well explained by its name (`live`). 

I'm also surprised to find that `live: false` and `live: undefined` **don't** do the same thing. I think that in every other instance I've seen of boolean props in the app, `undefined` is the same as `false`.

Finally, it's not claer that refresh interval is seconds (and it's often miliseconds in JS, right?), so maybe we could be clearer about that.

I think that on a personal note, I'd prefer to see this split up as something like: 

```suggestion
type TimeAgoProps = {
    date: Date | number | string | null | undefined;
    fallback?: string;
} & ({  dontUpdate?: boolean }
  | { refreshIntervalSeconds: number });
```

So that you avoid weird conflicts such as `dontUpdate={true} refreshIntervalSeconds={5}`, but also so that the default is still to update, as in the original code.

Optionally, we could take a cue from our SDKs and do something like this:

```suggestion
type TimeAgoProps = {
    date: Date | number | string | null | undefined;
    fallback?: string;
    refreshIntervalSeconds?: number
};
```

Where the default refresh interval is 12 (if left out), but you can set it to `0` to disable refreshing?

I'm happy to hear other suggestions here, though, I was just a little surprised by how this works.

Reply:
Good point. We should have it not configurable, using opinionated `12s` value. (60/5). It feels "alive" enough, we don't use seconds, it will be less then refetch interval on our SWR hooks.